### DISCUSSION TITLE
checkout@v1 broken

### DISCUSSION THREAD

**Discussion by rectalogic** [community/community#25829] [Actions]
<p>We can no longer checkout our repos that have private submodules. Previously we rewrote git@ urls to https:// and used a PAT and this worked. Recently checkout@v1 was updated and now seems incompatible with that approach.</p>
<p>I filed a bug here with details, but does anyone have a workaround? We have not figured out how to restore checkout functionality yet.</p>
<p><a href="https://github.com/actions/checkout/issues/60" rel="nofollow noopener">https://github.com/actions/checkout/issues/60</a></p>

**Comment by thboop** [comment, marked as answer]
<p>We are looking into this. </p>
<p>If you are encountering this issue, please use the previous minor version of the action</p>
<p><code>uses: actions/checkout@v1.0.0</code> </p>
<p>EDIT:</p>
<p>I’m happy to report the <code>uses: actions/checkout@v1</code> behavior around private submodules has been fixed! Please let me know if you see any more issues</p>
